geoips/plugins/modules/procflows/config_based.py

```diff
@@ -860,6 +860,10 @@ def call(fnames, command_line_args=None):
         presector_data = not command_line_args["no_presectoring"]
     else:
         presector_data = True
+    if command_line_args.get("output_checker_kwargs") is not None:
+        output_checker_kwargs = command_line_args["output_checker_kwargs"]
+    else:
+        output_checker_kwargs = {}
```

Comment:
I think this might be better done this way to avoid getting the same thing twice:
```
output_checker_kwargs = command_line_args.get("output_checker_kwargs", {})